### DISCUSSION TITLE
Damagedsprite is unaffected by character's ragdoll's texturescale

### DISCUSSION THREAD

**Discussion by TheNinjaScout** [FakeFishGames/Barotrauma#15201] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Damagedsprites aren't affected by ragdollparams' texturescale
Fixed in this PR: https://github.com/FakeFishGames/Barotrauma/pull/15202

### Reproduction steps

1. Set human's texturescale to 0.5
2. Spawn a human in-game and shoot them / damage them in one specific limb
3. Notice how the damagedsprite is bigger than the limb's texture it's supposed to be on
![image](https://github.com/user-attachments/assets/0eaf1571-360f-40de-87ef-97f7718b2602)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in our private repo, fix coming up in the next unstable build.
